sound/soc/sof/debug.c

```diff
@@ -588,6 +588,21 @@ int snd_sof_debugfs_io_item(struct snd_sof_dev *sdev,
 }
 EXPORT_SYMBOL_GPL(snd_sof_debugfs_io_item);
 
+int snd_sof_debugfs_add_region_item_iomem(struct snd_sof_dev *sdev,
```

Comment:
only minor. in the comit message " generic API...for generic DSP regions"? maybe a bit too generic :)

Reply:
@ranj063, yes, it is too generic, let me reword it ;)